### DISCUSSION TITLE
Realize shipping costs in Pretix

### DISCUSSION THREAD

**Discussion by blutonium** [pretix/pretix#3118] [Q&A] [answered]
Hi there,

we're selling merchandise with different variants (shirt sizes) via our pretix shop. For every order we need to add an amount of one of a product called "Shipping costs". At the moment, we're realizing this by adding an additional category to our merchandise product with an minimum and maximum amount of 1. The only item in this category is our "Shipping costs" item. This works fine for orders with only one merchandise item, but collapses when the amount rises. Currently the visitor needs to add the shipping costs for every product , which leads us to charge the shipping costs multiple times.

Is there any way to improve this?

**Comment by raphaelm** [comment, marked as answer]
We do offer a non-open-source plugin that supports this: https://marketplace.pretix.eu/products/shipping/

In the most simple cases, you might be able to emulate it with the (free) service fees plugin:
https://marketplace.pretix.eu/products/servicefees/

**Reply by blutonium** [reply]
Works fine for me, thank you!
